core/dbt/include/global_project/macros/materializations/models/table/columns_spec_ddl.sql

```diff
@@ -37,13 +37,32 @@
   {#--Obtain the column schema provided by the schema file by generating an 'empty schema' query from the model's columns. #}
```

Comment:
small nit (can't comment directly on the line) - we can remove 'order' from 'If any differences in name, data_type or order of columns' from the comment on line 32 above!